Config/LocustCavityEqCircuitTest.json

```diff
@@ -0,0 +1,48 @@
+{
+    "generators":
+    [          
+       "cavity-signal",
+       "lpf-fft",
+       "decimate-signal",
+       "digitizer"
+    ],
+
+    "cavity-signal":
+    {
+        "transmitter": "kass-current",
+	"e-gun": false,
```

Comment:
Looks good in general, but would suggest to maintain same level of indentation for readability.

Reply:
Sorry about that, I think my editor has a differently defined tab than the original author. I believe I've fixed it here to be consistent now.